tests/server/fastmcp/test_func_metadata.py

```diff
@@ -202,11 +203,8 @@ def func_dict_any() -> dict[str, Any]:
         return {"a": 1, "b": "hello", "c": [1, 2, 3]}
 
     meta = func_metadata(func_dict_any)
-    assert meta.output_schema == {
-        "additionalProperties": True,
-        "type": "object",
-        "title": "func_dict_anyDictOutput",
-    }
+
+    assert meta.output_schema == IsPartialDict(type="object", title="func_dict_anyDictOutput")
```

Comment:
This means that the `output_schema` has AT LEAST the keys I've added inside the `IsPartialDict`.